nameko/web/server.py

```diff
@@ -93,6 +93,7 @@ def start(self):
     def stop(self):
         self._is_accepting = False
         self._gt.kill()
+        self._sock.close()
```

Comment:
this is the listening socket, right? so calls in-flight can still finish and send responses?